packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenClassDoesNotExistAnymore.st

```diff
@@ -0,0 +1,23 @@
+as yet unclassified
+testSourceCodeWhenClassDoesNotExistAnymore
+
+	| messageTallyMock compiledMethod tree source |
+	
+	compiledMethod := (DummyClass>>#dummyMethodToProfile).
+	
+	messageTallyMock := MessageTallyMock new.
+	messageTallyMock method: compiledMethod.
+	 
+	tree := IPTTreeModelMock new.
+	tree currentSelection: messageTallyMock.
+		
+	DummyClass removeFromSystem.
+	source := tree selectedMessage.
+	
+	ClassBuilder new superclass: Object subclass: #DummyClass instanceVariableNames: '' classVariableNames: '' poolDictionaries: '' category: 'InteractiveProfilingTool-Tests'.
+	DummyClass compile: 'dummyMethodToProfile
+   	^ 1'.
+	
+	self assert: source equals: 'Missing'
```

Comment:
I'm starting to think that `IPTTreeModel >> selectedMessage` should raise an Error instead of returning `'Missing'`. What are your thoughts on this?

Reply:
Nach Absprache mit Eva haben wir uns dazu entschieden 'Missing' auszugeben.